case-study-azure-devops.md

```diff
@@ -0,0 +1,416 @@
+---
+title: Azure DevOps Case Study
+keywords: development platforms, Azure DevOps, Microsoft
+tags: Development Platforms, Azure DevOps, Microsoft
+sidebar: development-platforms
+permalink: development-platforms/case-study-azure-devops.html
+folder: development-platforms
+---
+
+## Table of contents
+
+1. [Introduction](#introduction)
+2. [Brief History](#brief-history)
+3. [Features](#features)
+4. [Strengths](#strengths)
+5. [Weaknesses](#weaknesses)
+6. [Pricing](#pricing)
+7. [Comparison](#comparison)
+8. [How To](#how-to)
+9. [Summary](#summary)
+10. [Credits](#credits)
+11. [References](#references)
+
+## **Introduction**
+
+<div style="width: 600px;">
+
+![Azure DevOps offerings](https://miro.medium.com/max/800/1*8orwInnxqPRhrcKf9aOo9Q.png)
+
+<p style="padding: 0; margin: 0;">
```

Comment:
```suggestion
```

This will apply to most inline html so I may stop suggesting after a while if there are a lot of these. HTML should be used sparingly in markdown for things like iframes that can't otherwise be done.